confluent_kafka/src/confluent_kafka.c

```diff
@@ -1131,6 +1131,37 @@ static void error_cb (rd_kafka_t *rk, int err, const char *reason, void *opaque)
 	CallState_resume(cs);
 }
 
+static void throttle_cb (rd_kafka_t *rk, const char *broker_name, int32_t broker_id,
+        int throttle_time_ms, void *opaque) {
```

Comment:
The indentation in the existing code is mixed up, but all new (and modified) C code shall use 8 whitespaces for tabs.

Reply:
ack